tests/test_volume.py

```diff
@@ -0,0 +1,20 @@
+import geoutils as gu
+
+import xdem
+
+
+def test_local_hypsometric():
+    xdem.examples.download_longyearbyen_examples(overwrite=False)
+
+    dem_2009 = gu.georaster.Raster(xdem.examples.FILEPATHS["longyearbyen_ref_dem"])
+    dem_1990 = gu.georaster.Raster(xdem.examples.FILEPATHS["longyearbyen_tba_dem"]).reproject(dem_2009)
+
+    outlines = gu.geovector.Vector(xdem.examples.FILEPATHS["longyearbyen_glacier_outlines"])
+    # Filter to only look at the Scott Turnerbreen glacier
+    outlines.ds = outlines.ds.loc[outlines.ds["NAME"] == "Scott Turnerbreen"]
+
+    mask = outlines.create_mask(dem_2009) == 255
+
+    ddem = dem_2009.data - dem_1990.data
+
+    ddem_bins = xdem.volume.hypsometric_binning(ddem.squeeze()[mask], dem_2009.data.squeeze()[mask])
```

Comment:
The tests should have an assert statement. In the case of sample data, you can check that the output array is equal to what you get today. Or at least a summary of it like mean, sum etc.

Reply:
Added some sanity checks in 115fdf6.